packages/shared-components/src/components/HeaderBar/HeaderBar.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+const HeaderBar = styled.div.withConfig({ displayName: 'HeaderBar' })`
+  height: 50px;
+  border-bottom: 1px solid #c2c2c2;
```

Comment:
Please use themeGet to use the theme value for this color and line thickness.

Reply:
ok, done